src/ArrayRateLimiter.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RateLimit;
+
+use RateLimit\Exception\LimitExceeded;
+
+final class ArrayRateLimiter implements RateLimiter, SilentRateLimiter
```

Comment:
I would name it `InMemoryRateLimiter`.

Reply:
The name is better. However, I developed the support this way to best match the implemented version of Redis.

Reply:
I renamed it to `InMemoryRateLimiter`.